build.gradle

```diff
@@ -2,25 +2,25 @@
 buildscript {
     repositories {
         google()
-        jcenter()
```

Comment:
Please revert - jcenter dependency should be first.

Reply:
(see above)